api/data_refinery_api/management/commands/post_downloads_summary.py

```diff
@@ -128,8 +128,8 @@ def should_display_email(email: str) -> bool:
         or email.startswith("d.prasad")
         or email.endswith("@alexslemonade.org")
         or email.endswith("@ccdatalab.org")
-        or email is ("daniel.himmelstein@gmail.com")
-        or email is ("dv.prasad991@gmail.com")
+        or email == ("daniel.himmelstein@gmail.com")
```

Comment:
Daniel doesn't work with us any longer and hasn't for some time - not sure it still makes sense to filter these if he turns out to be using it.